src/testing/stability_fr_parameter_range.h

```diff
@@ -0,0 +1,33 @@
+#ifndef __STABILITY_FR_PARAMETERS_RANGE_H__
+#define __STABILITY_FR_PARAMETERS_RANGE_H__
+
+#include "tests.h"
+#include "dg/dg.h"
+#include "parameters/all_parameters.h"
+
+namespace PHiLiP {
+namespace Tests {
+
+/// Burgers' periodic unsteady test
+template <int dim, int nstate>
+class StabilityFRParametersRange: public TestsBase
+{
+public:
+    /// Constructor
+    StabilityFRParametersRange(const Parameters::AllParameters *const parameters_input);
+
+    /// Destructor
+    ~StabilityFRParametersRange() {};
```

Comment:
Remove empty destructor to be consistent with https://github.com/dougshidong/PHiLiP/pull/243.